src/serializableStateInvariantMiddleware.test.ts

```diff
@@ -330,6 +330,61 @@ describe('serializableStateInvariantMiddleware', () => {
     expect(numTimesCalled).toBeGreaterThan(0)
   })
 
+  describe('ignored action paths', () => {
+    function reducer() {
+      return 0
+    }
+    const nonSerializableValue = new Map()
+
+    it('default value: meta.arg', () => {
```

Comment:
Let's throw in a use of `createAsyncThunk` here and dispatch one of its actions too, just to confirm that we that's consistent.

Reply:
I already added that over here: 

https://github.com/reduxjs/redux-toolkit/pull/457/files#diff-7ccfeb60873b37c2d4c0a8e4d255f298R466-R478